airbyte-server/src/main/java/io/airbyte/server/apis/ConnectionApiController.java

```diff
@@ -16,9 +16,13 @@
 import io.airbyte.server.handlers.ConnectionsHandler;
 import io.airbyte.server.handlers.OperationsHandler;
 import io.airbyte.server.handlers.SchedulerHandler;
+import io.micronaut.http.MediaType;
+import io.micronaut.http.annotation.Body;
+import io.micronaut.http.annotation.Controller;
+import io.micronaut.http.annotation.Post;
 import javax.ws.rs.Path;
```

Comment:
This import can be removed.

Reply:
Done, I think that I need to reconfigure the save action plugin